src/pages/dmt5/_download-app.tsx

```diff
@@ -83,142 +183,194 @@ const DownloadApp = () => {
+                                external="true"
+                                to={dmt5_app_store_url}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                <img src={AppStore} alt="google play" width="138" height="40" />
+                            </LocalizedLink>
+                        </DownloadLinkMobileWrapper>
                     </Flex>
                     <StyledHeader mt="2.4rem" as="h4">
                         {localize('Desktop')}
                     </StyledHeader>
-                    <Flex mt="0.8rem" jc="flex-start" height="auto">
+                    <StyledFlexMobile>
                         <DownloadLinkWrapper>
                             <LocalizedLink
                                 external="true"
-                                to={dmt5_windows_url}
+                                to={dmt5_linux_url}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                <img src={LinuxOs} alt="linux" />
+                            </LocalizedLink>
+                            <LocalizedLink
+                                external="true"
+                                to={dmt5_web_browser_url}
                                 target="_blank"
                                 rel="noopener noreferrer"
                             >
-                                <img src={Windows} alt="windows" />
+                                <img src={WebBrowser} alt="linux" />
+                            </LocalizedLink>
+                            <LocalizedLink
+                                external="true"
+                                to={dmt5_windows}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                <img src={Windows} alt="linux" />
                             </LocalizedLink>
                             <LocalizedLink
                                 external="true"
                                 to={dmt5_macos_url}
                                 target="_blank"
                                 rel="noopener noreferrer"
                             >
-                                <img src={MacOS} alt="macos" />
+                                <img src={MacOs} alt="macos" />
                             </LocalizedLink>
                         </DownloadLinkWrapper>
+                    </StyledFlexMobile>
+                </Mobile>
+            </StyledFlex>
+            <Desktop>
+                <StyledInfoContainer>
+                    <Separator />
+
+                    <StyledInfo src={MoreInfoDesktop} alt="more info" />
+                    <Text width="100%" ml="1.6rem" size="var(--text-size-m)" mt="96px">
```

Comment:
Please try to implement `Header` instead of `Text` in whole file, because in `src/components/elements/typography.js` it's marked as depricated 🙏 
I realize this has been here before but let's make our codebase better together 😃

Reply:
HI @NikitK-deriv , thanks for the comments, i have updated the code as per your suggestions. Please review🙏